samples/bluetooth/central_autoconn/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _bluetooth_central_autoconn:
+
+Bluetooth: Central Autoconnect
+##################
```

Comment:
This heading underline needs to be at least as long as the title above it, so add some more ``###``s